src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -50,10 +49,10 @@ internal sealed partial class AddressOfExpression : BaseAddressOfExpression, IAd
         public AddressOfExpression(IOperation reference, SemanticModel semanticModel, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue, bool isImplicit) :
             base(semanticModel, syntax, type, constantValue, isImplicit)
         {
-            ReferenceImpl = reference;
+            Reference = Operation.SetParentOperation(reference, this);
```

Comment:
Is `Operation.` qualifier necessary for these calls?

Reply:
I can remove those.